models/migrations/migrations.go

```diff
@@ -605,6 +605,8 @@ var migrations = []Migration{
 	NewMigration("Add Repository Licenses", v1_23.AddRepositoryLicenses),
 	// v306 -> v307
 	NewMigration("Add BlockAdminMergeOverride to ProtectedBranch", v1_23.AddBlockAdminMergeOverrideBranchProtection),
+	// v307 -> v308
+	NewMigration("Add index(user_id, is_deleted) for action table", v1_23.AddNewIndexForUserDashboard),
```

Comment:
I think we might need the unix CreatedUnix as the trailing data on the index the database would rely on temporary files to do the sorting. 

Another options is to change the queries that sort by CreatedUnix to use the PK as the ordering key. I think it would work since the ID is monotonic.

Reply:
That index does exist.